packages/core/src/checkbox/checkbox.ts

```diff
@@ -33,9 +36,15 @@ export class Checkbox extends FormControlMixin(HintMixin(LitElement)) {
   /** Whether the checkbox is checked. */
   @property({ type: Boolean, reflect: true }) checked = false;
 
+  /** Whether the checkbox is invalid. */
+  @property({ type: Boolean, reflect: true }) invalid = false;
```

Comment:
So if you set an initial value like this, it will trigger the Lit change detection for this property. If you instead do `invalid?: boolean;` it will still be falsy, but it won't trigger change detection. That can be useful if changing the property has side-effects. So I usually go with the `?: boolean;` version.